src/main/java/edu/greenblitz/robotName/commands/arm/Elbow/MoveElbowToAngle/MotorMoveElbowToAngle.java

```diff
@@ -0,0 +1,23 @@
+package edu.greenblitz.robotName.commands.arm.Elbow.MoveElbowToAngle;
+
+import edu.greenblitz.robotName.commands.arm.Elbow.ElbowCommand;
+
+public class MotorMoveElbowToAngle extends ElbowCommand {
```

Comment:
Is the name of the class like this because of simulated classes?

Even so, I think the name is kinda weird. Just `MoveElbowToAngle` is fine in my opinion

Reply:
but move is the conditional that pick simulation or motor